plugins/tds61.js

```diff
@@ -644,6 +644,9 @@ tds61 = {
         // Drop the FCSUBTYPE since we don't use it
         if (attrs.FCSUBTYPE) delete attrs.FCSUBTYPE;
 
+        // Drop the the "Not Found" F_CODE. This is from the UI
+        if (attrs.F_CODE == 'Not found') delete attrs.F_CODE;
+
```

Comment:
Hmmm, the MGCP schema should have the same code as the TDS:
```
        // Add the LayerName to the source
        if ((! tags.source) && layerName !== '') tags.source = 'mgcp:' + layerName.toLowerCase();
```

This is trying to account for the two different ways the translation is being called:
1) Shape/FGDB -> OSM:  We may not have a source tag (mapped to SDP in the spec) and if we don't, create the tag from the spec and the name of the layer. E.g. mgcp:lap030

2) At the end of the "from english" call from the UI:
  OSM -> MGCP -> "english" MGCP ->  User Edits -> "english" MGCP -> MGCP -> OSM
  The "MGCP -> OSM" part is the same calls as for 1. except that the layerName is empty.

Breaking the UI:  I don't think so.  This code is run as part of the TDS to OSM pre-processing. 
In TDS to OSM, we take "attrs" and return "tags". Since this F_CODE value ("Not found"), would cause issues during the translation, it is easier to drop it than deal with the warnings and errors.